pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/EndEnrollmentSupportESC.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+import lombok.extern.slf4j.Slf4j;
+import org.jdbi.v3.core.Handle;
+
+@Slf4j
+public class EndEnrollmentSupportESC extends EndStudyEnrollmentSupport {
+
+    public EndEnrollmentSupportESC() {
+        super("cmi-esc", "FOLLOWUPCONSENT");
+        log.info("TASK:: EndEnrollmentSupportESC ");
+    }
+
+    @Override
+    public void run(final Handle handle) {
```

Comment:
It does not look like this override does anything?

Reply:
cleaned it up